bciers/apps/compliance/src/tests/components/compliance-summary/request-issuance/internal/track-status-of-issuance/InternalTrackStatusOfIssuanceComponent.test.tsx

```diff
@@ -88,32 +88,6 @@ describe("InternalTrackStatusOfIssuanceComponent", () => {
     ).toBeNull();
   });
 
-  it("does not show Analyst's Comment for analyst route when auto-declined due to supplementary report", () => {
```

Comment:
For better test coverage, we need to add the two tests below:
- `shows Analyst's Comment for CAS_ANALYST role when auto-declined due to supplementary report`
- `hides Analyst's Comment for non-internal reviewer role even when supplementary report required`